WordClock/Display.cpp

```diff
@@ -142,6 +142,19 @@ void Display::setBrightness(uint8_t brightness) {
   pixels.show();
 }
 
+void Display::setColor(uint8_t r, uint8_t g, uint8_t b) {
+  ClockConfig.clockColor = pixels.Color(r, g, b);
```

Comment:
Wanneer Config::save() aangeroepen wordt zal de geselecteerde kleur als 'default' opgeslagen worden in het EEPROM. Is dat wat je wilde bereiken?

Reply:
Klopt!